d2-schemas/src/main/pegasus/com/linkedin/d2/D2TransportClientProperties.pdl

```diff
@@ -141,4 +141,9 @@ record D2TransportClientProperties {
      */
     HTTP_2
   }
+
+  /**
+   * Maximum ratio of retry requests to total requests per client
+   */
+  maxClientRequestRetryRatio: optional double
```

Comment:
The code is setting a default of 0.2 for this, any reason why it cannot be added in the schema?

Reply:
As with other D2 config properties defined in the schema, the default value is defined in two places: in Pegasus factory and in global d2.src config. So the default value can then be easily discovered by D2 config services (e.g. d2-config-service, maestro), and also be tuned easily if needed. We don't wan't to define the it directly in the schema because it creates an additional source of truth of the default value.